### PR TITLE
[13258] Release notes v0.4.0

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Signed-off-by: JLBuenoLopez-eProsima <joseluisbueno@eprosima.com>

### CODE REVIEW

**Comment by JLBuenoLopez-eProsima** [comment]
CI is failing due to an issue which is explained and solved in #140. This PR should be rebased after that PR is merged.

**Comment by JLBuenoLopez-eProsima** [comment]
CI failures are unrelated

File: docs/rst/notes/notes.rst [hunk 1]
```diff
@@ -1,35 +1,26 @@
...
-* By default, statistics data is received using UDP transport, shared memory is disabled.
-* Network latency data now relates to a source participant and a destination locator
-  (previously a source locator and a destination locator).
-* Statistics data can now trigger the discovery of a new locator.
-* Improved entity names and aliases to be more user friendly.
-* Solved an issue that may cause the internal database to freeze.
-* Allow for topics with the same name to be on different domains.
+* Return the end of the time interval as the data point's timestamps instead of the initial one.
+* Avoid adding the same locator twice to the database.
+* Fix deadlock when accessing the database within a callback implementation.
```
Comment (MiguelCompany) [inline docs/rst/notes/notes.rst:19]:
```suggestion
* Fix deadlock when accessing the database within a callback implementation.
* Avoid using deprecated namespace.
* Improvements on CI
```

**Review by MiguelCompany — CHANGES_REQUESTED** [review]

**Review by MiguelCompany — DISMISSED** [review]

**Review by MiguelCompany — APPROVED** [review]
👍

**Pull request merged**